zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/metrics/GatewayMetrics.java

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.zeebe.gateway.metrics;
-
-import io.prometheus.client.Counter;
-import io.prometheus.client.Histogram;
-
-public final class GatewayMetrics {
-
-  private static final Histogram REQUEST_LATENCY =
```

Comment:
It's a shame they were not in use anymore, could be useful when debugging latency